tests/cli/component-model/shared-everything-threads/not-accepted.wast

```diff
@@ -1,20 +1,34 @@
 ;; RUN: wast --assert default --snapshot tests/snapshots % -f shared-everything-threads
 
-(assert_invalid
-  (component
-    (core module $A
-      (memory (export "m") 1 2 shared))
-    (core instance $A (instantiate $A))
-    (alias core export $A "m" (core memory $m))
+(component
+  (core module $A
+    (memory (export "m") 1 2 shared))
+  (core instance $A (instantiate $A))
+  (alias core export $A "m" (core memory $m))
+
+  (core module $B (import "" "" (memory 1 2 shared)))
+  (core instance (instantiate $B (with "" (instance (export "" (memory $m))))))
+)
+
+(component
+  (core module $A
+    (table (export "m") shared 1 2 (ref null (shared func)))
   )
-  "shared linear memories are not compatible with components yet")
+  (core instance $A (instantiate $A))
+  (alias core export $A "m" (core table $m))
+
+  (core module $B (import "" "" (table shared 1 2 (ref null (shared func)))))
+  (core instance (instantiate $B (with "" (instance (export "" (table $m))))))
+)
```

Comment:
You want to keep these tests in `not-accepted.wast`? Or rename this file? Not a big deal if we leave things here, I suppose, just noting the file name.

Reply:
I tend to pay far less attention to filenames than I should unfortunately... I think I'll go ahead and merge this PR, but I think it'd be best to clean up some organization from the old `tests/local/*` test suite which is no longer necessary in the future